examples/louse 2.nwk

```diff
@@ -0,0 +1 @@
+((p18, p19) p4, ((p22, (p24, p25) p23) p20, (p26, ((p30, p31) p28, (p32, p33) p29) p27) p21) p5) p3
```

Comment:
Duplicate -- it seems like these duplicates might have been created my merge? Please be a little careful when you merge.

Reply:
Removed.